CONTRIBUTING.md

```diff
@@ -34,3 +34,36 @@ Guidelines](https://opensource.google/conduct/).
 All submissions, including submissions by project members, require review. We
 use [GitHub pull requests](https://docs.github.com/articles/about-pull-requests)
 for this purpose.
+
+## For Maintainers
+
+### Preparation
+
+Install the pre-push hook:
+```
+cp devtools/pre-push-hook.sh .git/hooks/pre-push
+```
+
+### Creating a new release
+
+This repo consists of a single Go module.
+To increase the minor or patch version of the module:
+
+1. Determine the desired tag, using `git tag -l` to see existing tags
+   and incrementing as appropriate. We will call the result TAG in
```

Comment:
Mention that TAG naming should be `vX.Y.Z`

Reply:
Done.